src/System.IO.FileSystem/src/System/IO/FileSystem.Windows.cs

```diff
@@ -328,7 +328,7 @@ public static DateTimeOffset GetLastWriteTime(string fullPath)
 
         public static void MoveDirectory(string sourceFullPath, string destFullPath)
         {
-            if (!Interop.Kernel32.MoveFile(sourceFullPath, destFullPath))
+            if (!Interop.Kernel32.MoveFile(sourceFullPath, destFullPath, false))
```

Comment:
Call a shared helper- you want the new overload to behave in exactly the same way if you pass false.